Tiltfile

```diff
@@ -5,7 +5,7 @@ docker_build(
   'api-image',
   context='.',
   dockerfile='./Dockerfile.dev',
-  ignore=['./node_modules', './.infra'],
+  ignore=['./node_modules', './.infra', '__tests__'],
```

Comment:
Not sure if it is needed, it just kept on rebuilding whenever I did a change in tests (often)

Reply:
Perfect!